File: src/main/java/dkoryakin/aston/demo/api/body/request/AccountDepositPostBody.java
package dkoryakin.aston.demo.api.body.request;


import jakarta.validation.constraints.NotNull;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AccountDepositPostBody {

    @NotNull
    private BigDecimal amount;

}

File: src/main/java/dkoryakin/aston/demo/api/body/AccountDepositPostBody.kt
package dkoryakin.aston.demo.api.body

import dkoryakin.aston.demo.infrastructure.validator.PinCodeConstraint
import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.NotNull
import org.hibernate.validator.constraints.Length


data class AccountDepositPostBody(@NotNull val amount:Double) {
}